data_ocean/views.py

```diff
@@ -67,6 +67,11 @@ class DOOpenAPISchemaGenerator(OpenAPISchemaGenerator):
             'Зручний доступ до даних за допомогою Rest API для розробників програмного забезпечення.<p/>'
         ),
         contact=openapi.Contact(email="info@dataocean.us"),
+        x_logo={
+            'url': f'{settings_local.BACKEND_SITE_URL}/static/logo.png',
```

Comment:
you already on backend...
```suggestion
            'url': f'{settings.STATIC_URL}logo.png',
```